RadiationForce_using_FIR_Filter/compare_runs.m

```diff
@@ -0,0 +1,106 @@
+% This script runs the RM3 example and compares the accuracy and run-times
+% when the radiation force is calculated using the radiation convolution 
+% method, the FIR filter method, and the state-space realization.
+clc;close all;clear;
+
+% Baseline case with no-convolution for Radiation Force calculations,
+
+enable_convolution = 0;
+enable_FIR         = 0;
+enable_ss          = 0;
+t_regular          = tic;
+wecSim
+run_time.baseline    = toc(t_regular);
+run_summary.baseline = output;
+clearvars -except run_summary run_time
+
+
+% FIR for Radiation Force calculations,
+
+enable_convolution = 0; 
+enable_FIR         = 1;
+enable_ss          = 0;
+tStart_FIR = tic;
+wecSim
+run_time.FIR    = toc(tStart_FIR);
+run_summary.FIR = output;
+clearvars -except run_summary run_time
+
+
+% State-Space for Radiation Force calculations,
+
+enable_convolution = 0; 
+enable_FIR         = 0;
+enable_ss          = 1;
+tStart_ss          = tic;
+wecSim
+run_time.state_space   = toc(tStart_ss);
+run_summary.state_space = output;
+clearvars -except run_summary run_time
+
+% Convolution for Radiation Force calculations,
+
+enable_convolution = 1; 
+enable_FIR         = 0;
+enable_ss          = 0;
+t_conv             = tic;
+wecSim
+run_time.convolution    = toc(t_conv);
+run_summary.convolution = output;
+clearvars -except run_summary run_time
+
+
+
+
+
+%% Plot time-history
+mode = 3;
+pos_baseline    =  run_summary.baseline.bodies(1).position;
+pos_convolution =  run_summary.convolution.bodies(1).position ;
+pos_FIR         =  run_summary.FIR.bodies(1).position;
+pos_state_space =  run_summary.state_space.bodies(1).position;
+h(1) = figure();
+y_data = pos_baseline(:,mode); y_name = 'Position (m)';
+x_data =  run_summary.baseline.bodies(1).time  ; x_name = 'Time (s)';
+Title  =  ''; FS = 14; LW = 2;style = '-'; color = 'r';
```

Comment:
@salhus Very minor comment: Personal preference, so ignore if desired, but request the Style option use '--', '-.', and ':' to differentiate lines in addition to color.

Reply:
@nathanmtom 
- done